src/mesido/financial_mixin.py

```diff
@@ -1266,7 +1272,7 @@ def __revenue_constraints(self, ensemble_member):
                 nominal = self.variable_nominal(variable_revenue_var)
 
                 sum = 0.0
-                timesteps = np.diff(self.times()) / 3600.0
+                timesteps = np.diff(self.times()) * multiplier  # / 3600.0
```

Comment:
1.  it seemed that sometimes the costs were in EUR/Wh while other times it was in EUR/g (electricty vs gas), this would mean, that for the first only timesteps in hours are required while for gas timesteps in seconds because massflow is g/s.
2. I'll change it to cost_multiplier.
3. I'll delete it.